cmd/main.go

```diff
@@ -69,11 +70,14 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.DurationVar(&requeueTime, "requeue-time", RequeueDuration,
-		"Duration for the validation reconciliation loop. "+
+		"The maximum times it takes between reconciliations of DNS Record "+
 			"Controls how ofter record is reconciled")
 	flag.DurationVar(&validFor, "valid-for", ValidityDuration,
 		"Duration when the record is considered to hold valid information"+
 			"Controls if we commit to the full reconcile loop")
+	flag.DurationVar(&validFor, "validation-duration", DefaultValidationDuration,
```

Comment:
this looks like a dupe of the line above?

Reply:
wrong war was holding the value. Fixed